db/migrate/20171219091048_add_token_hash_to_invitation_token.rb

```diff
@@ -0,0 +1,6 @@
+class AddTokenHashToInvitationToken < ActiveRecord::Migration[5.0]
+  def change
+    add_column :invitation_tokens, :token_hash, :string
```

Comment:
hier moeten we denk ik nog even iets bij zetten voor het aanmaken van `token_hash`s